### PR TITLE
Document session lifetime handling

### PR DESCRIPTION
**Pull request by jms-pantheon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,20 @@ https://github.com/pantheon-systems/wp-native-php-sessions
 
 That's it!
 
+## Configuration ##
+
+By default the session lifetime is set to 0, which is until the browser is closed. 
+
+To override this use the "pantheon_session_expiration" filter before the WordPress Native PHP Sessions plugin is loaded. For example a small Must-use plugin (a.k.a. mu-plugin) could contain:
```
Comment (CdrMarks) [inline README.md:37]:
```suggestion
To override this use the `pantheon_session_expiration` filter before the WordPress Native PHP Sessions plugin is loaded. For example a small Must-use plugin (a.k.a. mu-plugin) could contain:
```

File: README.md [hunk 2]
```diff
@@ -30,6 +30,20 @@ https://github.com/pantheon-systems/wp-native-php-sessions
...
+
+```
+<?php
+function my_session_expiration_override() {
+    return 60*60*4; // 4 hours
+}
+add_filter( 'pantheon_session_expiration', 'my_session_expiration_override');
```
Comment (CdrMarks) [inline README.md:44]:
```suggestion
add_filter( 'pantheon_session_expiration', 'my_session_expiration_override' );
```

**Review by CdrMarks — CHANGES_REQUESTED** [review]
Let me know what you think about the suggestions I made.

**Review by CdrMarks — DISMISSED** [review]

**Comment by jspellman814** [comment]
Please make this PR against develop as per [CONTRIBUTING.md](https://github.com/pantheon-systems/wp-native-php-sessions/blob/main/CONTRIBUTING.md#workflow)

**Review by jspellman814 — CHANGES_REQUESTED** [review]
This text should also be added to [readme.txt](https://github.com/pantheon-systems/wp-native-php-sessions/blob/main/readme.txt) so it appears on WP.org. 

There is [automation in place](https://github.com/pantheon-systems/wp-native-php-sessions/blob/main/package.json#L12) to handle this (though that goes from readme.txt to README.md and you've updated the .md file), or you can copy over manually

**Comment by jms-pantheon** [comment]
Thanks for that review @jspellman814 I have set this PR to be based on Develop and made the change to readme.txt

It seems like Markdown is only partially supported on WordPress.org for readme.txt files, I didn't find any examples of code blocks. 

Would it be better to include this as just straight text block, or as code with each line with its single back-ticks?

https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/

**Comment by jspellman814** [comment]
@jms-pantheon We have some code in the [solr-power readme.txt](https://github.com/pantheon-systems/solr-power/blob/main/readme.txt#L96). Here's the rendering on wp.org: https://wordpress.org/plugins/solr-power/

**Comment by jms-pantheon** [comment]
Wow thank you, I searched really hard for that one @jspellman814 but couldn't find an example. I've changed this to use the same 4 space indent.

**Review by jspellman814 — APPROVED** [review]
LGTM! Thanks @jms-pantheon 👍

**Pull request merged**